samples/tests/knowledge_base_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+import os
+
+from .. import knowledge_base_management
+from .. import document_management
+from .. import detect_intent_knowledge
+
+PROJECT_ID = os.getenv('GCLOUD_PROJECT')
+SESSION_ID = 'fake_session_for_testing'
+TEXTS = ['Where is my data stored?']
+
+KNOWLEDGE_BASE_NAME = 'fake_knowledge_base_name'
+DOCUMENT_BASE_NAME = 'fake_document_name'
+
+
+def test_create_knowledge_base(capsys):
+    # Create a knowledge base
+    knowledge_base_management.create_knowledge_base(PROJECT_ID,
```

Comment:
first confirm there isn't already a knowledge base with the name, then create, then assert that there now is a knowledge base of that name.